apps/designer/app/designer/features/style-panel/shared/property-icon.tsx

```diff
@@ -0,0 +1,17 @@
+import { TextFieldIconButton } from "@webstudio-is/design-system";
+import { useIsFromCurrentBreakpoint } from "./use-is-from-current-breakpoint";
+import type { PropertyProps } from "../style-sections";
+import { forwardRef } from "react";
+
+export const PropertyIcon = forwardRef<
+  HTMLButtonElement,
+  PropertyProps & { children: JSX.Element }
+>(({ property, children }) => {
+  const isCurrentBreakpoint = useIsFromCurrentBreakpoint(property);
+  return (
+    <TextFieldIconButton state={isCurrentBreakpoint ? "breakpoint" : undefined}>
```

Comment:
2 problems:
1. We also need to show the resize icon for the scrub
2. TextFieldIconButton isn't really aware of feature specific concepts like a breakpoint, so you are leaking abstraction now.

We need to solve this  by making CssValueInput control both of those things, since its the only one aware of both.

Do you know if we are going to need PropertyIcon anywhere else than CssValueInput? If no - this can be moved into css-value-input folder.